### ISSUE TITLE
Total loss and mask_cls_loss  are the same during the 2 and 3 training stage

### ISSUE THREAD

**Issue by whrenstone** [matterport/Mask_RCNN#250]
Hi!  @waleedka I am trying to train the project from scratch using coco 2014 dataset  to get the same recognition result like the weights (mask_rcnn_coco.h5). But i am encountered 2 problems mainly:
       1, the trained network can only recognize person correctly, while the other objects are wrongly classified ,for example: cars in the val images are labeled as motorcycles; 
many other instances are classified as person.
       2.during the training procedure, the total loss dropped dramatically in the 
1 stage. But the loss' descent stopped in the 2 and 3 stage, and the mask_cls_loss is about 0.7 while the total loss is 2.4. I have tried to change learning_rate from 2/1000 to 5/10000 but nothing happended.
	The parameters i used  are basically the same as you provided，and i  found 
that in coco dataset，person instances number are much more than the others instances. So  i am trying to manually decrease the person instances for network's training to keep all instances number balanced. 
        I really appreciate any suggestions from yours about howto reduce training total losses or mask_cls_losses! Many Thanks!

**Comment by waleedka** [comment]
Are you training from imagenet weights? Or really, from scratch? I think it would be really hard to train it from scratch.

**Comment by whrenstone** [comment]
@waleedka  Yes, i am trying to train the network from scratch to understand the whole procedure. It seems  
the network recognize person first, and cars. But the false positive rate is high and the mask class loss remains the same whatever the learning rate is. Anyway, thanks for your reply and brilliant codes!

**Comment by FruVirus** [comment]
@whrenstone 

Are you training the entire Mask R-CNN in stages or the whole thing at once?
